### PR TITLE
Add FAQ about frameworks in the single-spa-config

### PR DESCRIPTION
**Pull request by TheMcMurder**
I think this is a common point of confusion and I've noticed an increase in questions about frameworks in the root config.

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -13,6 +13,9 @@ The applications can all be written in the same framework, or they can be implem
 ## Is there a recommended setup?
 Yes, here is [the documentation for our recommended setup](/docs/recommended-setup/).
 
+## Should I put frameworks in the single-spa-config/root-application?
```
Comment (frehner) [inline docs/faq.md]:
```suggestion
## Should I have a parent/root app and children apps?
```

File: docs/faq.md [hunk 2]
```diff
@@ -13,6 +13,9 @@ The applications can all be written in the same framework, or they can be implem
...
+No. We strongly encourage that your single-spa-config or root application does not use any javascript ui-frameworks (React, Angular, Angularjs, Vue, etc). In our experience a plain javascript module is best for the single-spa-config and only the registered applications actually use ui-frameworks (angular, react, vue, etc).
```
Comment (frehner) [inline docs/faq.md:17]:
```suggestion
No. We strongly encourage that your single-spa-config or root application does not use any javascript ui-frameworks (React, Angular, Angularjs, Vue, etc). In our experience a plain javascript module is best for the single-spa-config and only the registered applications actually use ui-frameworks (angular, react, vue, etc). 

Why? You end up creating a structure that has all the disadvantages of microservices without any of the advantages: your apps are now coupled together and you have to change multiple apps at the same time in order to make updates. Good microservices are completely **independent**, and this pattern breaks that.
```

**Review by frehner — COMMENTED** [review]
feel free to take/throw out my suggestions here.

**Review by frehner — APPROVED** [review]
🐽

**Comment by internettrans** [comment]
Oo yes I completely agree

**Pull request merged**